Rakefile

```diff
@@ -9,3 +9,32 @@ Rails.application.load_tasks
 
 desc "Run all the tests"
 task default: %i[spec standard]
+
+namespace :app do
+  desc "Generate static pages"
+  task static: :environment do
+    Rails.application.config.action_controller.perform_caching = false
+    pages = {
+      "pages/errors/404" => "404.html",
+      "pages/errors/422" => "422.html",
+      "pages/errors/500" => "500.html",
+      # TODO: Generate this page, inlining all assets (CSS, JS, and images as data URLs)
```

Comment:
This looks like it could be a reference to an issue

Reply:
It could. For context, we're in the finishing off stages here and I'm not sure we know how we want to track this sort of thing with the client ongoing.